src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheck.java

```diff
@@ -807,13 +807,12 @@ public final void setSortImportsInGroupAlphabetically(boolean value) {
      *        user value.
      */
     public final void setCustomImportOrderRules(final String inputCustomImportOrder) {
-        if (!customImportOrderRules.equals(inputCustomImportOrder)) {
+        if (!CUSTOM_IMPORT_ORDER_RULES.equals(inputCustomImportOrder)) {
```

Comment:
I don't get it :) Why do we need `CUSTOM_IMPORT_ORDER_RULES` when we just compare the input with empty string in this place only?

Reply:
We have fields as constants for default values of all users accessible properties.
 But I agree it looked weird even for me.

If expression as inputCustomImportOrder.isEmpty will looks better. 

Naming constant as DEFAULT_xxxxxx would make it more clear, but all other default value constants are not named like this. Should we rename all of them to be more explicit?

But end up in gray area of ... What is default value of property for this module, as for user it is String.
https://checkstyle.org/config_imports.html#CustomImportOrder_Properties

Reply:
I see, so it's a Checkstyle's requirement that the user's property field must be present even though it is not actually used. This allows setter to be called and use incoming value in the way we want.

I don't see we use DEFAULT_xxx that much in other checks, but I agree that `DEFAULT_CUSTOM_IMPORT_ORDER_RULES.equals(inputCustomImportOrder)` is more readable than `CUSTOM_IMPORT_ORDER_RULES.equals(inputCustomImportOrder)`

Reply:
Ok then I have changed name as `DEFAULT_CUSTOM_IMPORT_ORDER_RULES`

Reply:
It is not a requirement, but end up our pattern for most of Checks.

Reply:
@rdiachenko @romani ping